src/stdio.rs

```diff
@@ -0,0 +1,243 @@
+use super::Error;
+
+use core::mem::ManuallyDrop;
+
```

Comment:
```suggestion
```

Reply:
These could be automatically applied by `cargo fmt` though.